WordPress/twentytwentyfive
# Try: Remove wrappers from landing pages, and set zero margin to individual patterns.

**Description**

Fixes https://github.com/WordPress/twentytwentyfive/issues/664

We're removing the wrappers of the landing pages, so that they're functional in the zoom out view. To respect the designs, we are also setting a zero margin to the patterns that are involved in these landing pages.

**Screenshots**



https://github.com/user-attachments/assets/9dde23f0-0844-4e21-a565-ab01c0e4d8e8


**Testing Instructions**

1. Create a page.
2. Insert a landing page. Choose from: "Shop homepage", "Business homepage", "Landing page for book", "Landing page for event", "Landing page for podcast".
3. Confirm that the design is ok.
4. Confirm that the styles of the main sections of the landing pages are accessible in zoom out view.

Repeat again, creating a page from the site editor.
